src/Functions/FunctionsStringDistance.cpp

```diff
@@ -181,7 +217,9 @@ REGISTER_FUNCTION(StringDistance)
 
     factory.registerFunction<FunctionByteEditDistance>(
         FunctionDocumentation{.description = R"(Calculates the edit distance between two byte-strings.)"});
-
     factory.registerAlias("levenshteinDistance", NameEditDistance::name);
+
+    factory.registerFunction<FunctionByteJaccardIndex>(
```

Comment:
The Levenshtein alias (l. 220) isn't documented in official docs. Same for "mismatches" (l. 216).